scipy/spatial/transform/rotation.py

```diff
@@ -1858,7 +1931,7 @@ def match_vectors(cls, a, b, weights=None):
                              "vector.")
```

Comment:
This check is not necessary. We can estimate a rotation and calculate the RMSD. The warning about a poorly defined rotation will be raised, but this is valid input.